design/ceph/ceph-config-updates.md

```diff
@@ -135,7 +136,8 @@ spec:
       debug_osd: 10
    # ...
 ```
-**Note on the above:** all values under config are reported to Ceph as strings, but the yaml should
+!!! Note
+All values under config are reported to Ceph as strings, but the yaml should
```

Comment:
To render properly, callouts need to have their text indented 4 spaces (to the same level as the type of callout).

```suggestion
!!! Note
    All values under config are reported to Ceph as strings, but the yaml should
```